### ISSUE TITLE
Replace pagination code from utility function paginated_queryset in hosts app

### ISSUE THREAD

**Issue by TroJan** [Cloud-CV/EvalAI#445]
`paginated_queryset` is a function in `base utils.py`.

Idea is to remove paginator code with a function named `paginated_queryset`

```
paginator = PageNumberPagination()
paginator.page_size = settings.REST_FRAMEWORK['PAGE_SIZE']
result_page = paginator.paginate_queryset(participant_teams, request)
```

For reference, look at PR #422

**Comment by jainarunarunjain** [comment]
@deshraj @TroJan  Can i work on this

**Comment by deshraj** [comment]
Sure, go ahead. Assigning it to you.

**Comment by TroJan** [comment]
Fixed in #450
